notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>No "floating" figure: figures should always be referenced in the text. Maybe (1), (2), (3) could be referenced in the definitions 1-3 above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>good catch! i added a description of the figure above: The figure below provides a visualization of each of these definitions. The first figure is a visualization of "local parametrization", the second is a visualization of "local implicit function" and the third is a visualization of "local graph":</p>